static/js/components/email/constants.js

```diff
@@ -0,0 +1,2 @@
+export const SEARCH_EMAIL_TYPE = 'searchResultEmail';
+export const COURSE_EMAIL_TYPE = 'courseTeamEmail';
```

Comment:
it's a string constant, so i think it's appropriate to represent it with caps+snake case (we also do this in `static/js/constants.js`)